<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>_MainWindow</class>
 <widget class="QMainWindow" name="_MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>617</width>
    <height>467</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RTKUSER</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../../../../horus/rtk2csv/icon/icon.ico</normaloff>../../../../../../../horus/rtk2csv/icon/icon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="1">
         <widget class="QLineEdit" name="baseRawInput">
          <property name="placeholderText">
           <string>Dados da Base</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="baseRawButton">
          <property name="text">
           <string>Open File</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="hoverRawInput">
          <property name="placeholderText">
           <string>Dados do Rover</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="hoverRawButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Open File</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Base:   </string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Rover:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Antenna height (m):</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="AntAltInput">
          <property name="text">
           <string>2.104</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QCheckBox" name="checkIBGE">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>IBGE</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="1" column="0">
           <widget class="QLabel" name="label_4">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Nav</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="IBGEObsInput">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="placeholderText">
             <string>Observação</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="IBGENavInput">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="placeholderText">
             <string>Navegação</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_3">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Obs</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="IBGEObsButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Open File</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="IBGENavButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Open File</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QCheckBox" name="checkBaseRtk">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Base RTK</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="baseLatInput">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>Lat</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="baseLonInput">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="placeholderText">
           <string>Lon</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="baseAltInput">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="placeholderText">
           <string>Alt</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="basePositionButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Open File</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTextEdit" name="output"/>
      </item>
      <item>
       <widget class="QPushButton" name="processButton">
        <property name="text">
         <string>Process</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
